controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -706,38 +743,51 @@ func (r *StorageClassClaimReconciler) reconcileCephBlockPool() error {
 
 	capacity := r.storageConsumer.Spec.Capacity.String()
 
+	cephClients := map[string]string{
+		"provisioner": r.cephClientProvisioner.Name,
+		"node":        r.cephClientNode.Name,
+	}
+
```

Comment:
Why was this moved here? can we keep it in its original code line? I would prefer to define vars as close to usage as possible.

Reply:
its was left over when i stored blockpool name in the CR status, placing at previous line